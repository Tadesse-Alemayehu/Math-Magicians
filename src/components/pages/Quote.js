import React from 'react';

export default function Quote() {
  return (
    <div>
      <h1 className="main-heading">Math Magicians</h1>
      <h2 className="quote-text">
        Mathematics is not about numbers, equation, computations
        <br />
        or algorithms. It is about understandin. - William paul Thurston
      </h2>
    </div>
  );
}
