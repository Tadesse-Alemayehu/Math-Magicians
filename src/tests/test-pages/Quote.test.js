import renderer from 'react-test-renderer';
import Quote from '../../components/pages/Quote';

it('should render Quotes component', () => {
  const quote = renderer.create(<Quote />).toJSON();
  expect(quote).toMatchSnapshot();
});
