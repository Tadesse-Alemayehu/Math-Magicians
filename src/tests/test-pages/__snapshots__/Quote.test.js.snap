// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Quotes component 1`] = `
<div>
  <h1
    className="main-heading"
  >
    Math Magicians
  </h1>
  <h2
    className="quote-text"
  >
    Mathematics is not about numbers, equation, computations
    <br />
    or algorithms. It is about understandin. - William paul Thurston
  </h2>
</div>
`;
